speech/google/cloud/speech/client.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, credentials=None, _http=None, _use_grpc=None):
             self._use_grpc = _use_grpc
 
     def sample(self, content=None, source_uri=None, stream=None, encoding=None,
-               sample_rate=None):
+               sample_rate_hertz=None):
```

Comment:
Added `hertz` here just for our library but the name in the protobuf hasn't changed?